javaparser-core/src/main/java/com/github/javaparser/ast/expr/VariableDeclarationExpr.java

```diff
@@ -47,106 +49,140 @@
 
     private EnumSet<Modifier> modifiers = EnumSet.noneOf(Modifier.class);
 
-	private List<AnnotationExpr> annotations;
-
-	private Type type;
-
-	private List<VariableDeclarator> vars;
-
-	public VariableDeclarationExpr() {
-	}
-
-	public VariableDeclarationExpr(final Type type, final List<VariableDeclarator> vars) {
-		setType(type);
-		setVars(vars);
-	}
+    private List<AnnotationExpr> annotations;
+
+    private Type type;
+
+    private List<VariableDeclarator> vars;
+
+    public VariableDeclarationExpr() {
+    }
+
+    public VariableDeclarationExpr(final Type type, String variableName) {
+        setType(type);
+        List<VariableDeclarator> vars = new ArrayList<>();
+        vars.add(new VariableDeclarator(variableName));
+        setVars(vars);
+    }
+
+    public VariableDeclarationExpr(final Type type, VariableDeclarator var) {
+        setType(type);
+        List<VariableDeclarator> vars = new ArrayList<>();
+        vars.add(var);
+        setVars(vars);
```

Comment:
Good point, I tend not to use this as it's a bit of an obscure method, being able to use it with an array or values isn't right to me.
I'll rework this